LiteCore/Support/Logging.cc

```diff
@@ -179,7 +179,7 @@ namespace litecore {
         if(encoder) {
             auto newEncoder = new LogEncoder(*sFileOut[(int)level], level);
             sLogEncoder[(int)level] = newEncoder;
-            newEncoder->log("", LogEncoder::None, "---- %s ----", sInitialMessage.c_str());
+            newEncoder->log("", map<unsigned, string>(), LogEncoder::None, "---- %s ----", sInitialMessage.c_str());
```

Comment:
Tip: you can simplify that new parameter to just `{}`. (No need to change it here unless you want to.)